docs/src/reference/variables.md

```diff
@@ -49,4 +49,15 @@ num_variables
 VariableRef
 index(::VariableRef)
 optimizer_index(::VariableRef)
+
+add_variable
+check_belongs_to_model
+build_variable
+parse_one_operator_variable
```

Comment:
This function is currently exported. Should it be?

Reply:
Yes: https://github.com/blegat/SetProg.jl/blob/64c056f8c22b3b79fdc0862c8938a0f23bd03796/src/macros.jl